src/components/clusterConfiguration/ReviewStep.tsx

```diff
@@ -9,12 +9,21 @@ import ClusterWizardToolbar from '../clusterWizard/ClusterWizardToolbar';
 import { getErrorMessage, handleApiError, postInstallCluster } from '../../api';
 import { updateCluster } from '../../features/clusters/currentClusterSlice';
 import ReviewCluster from './ReviewCluster';
+import { useOpenshiftVersions } from '../fetching/openshiftVersions';
+import LoadingState from '../ui/uiState/LoadingState';
 
 const ReviewStep: React.FC<{ cluster: Cluster }> = ({ cluster }) => {
   const { addAlert } = React.useContext(AlertsContext);
   const { setCurrentStepId } = React.useContext(ClusterWizardContext);
   const dispatch = useDispatch();
 
+  const { error: errorOCPVersions, loading: loadingOCPVersions, versions } = useOpenshiftVersions();
+
+  React.useEffect(() => errorOCPVersions && addAlert(errorOCPVersions), [
```

Comment:
I think we should move that higher in the components hierarchy. The OCP versions are used/retrieved on other steps as well, there's no need to retrieve them repeatedly. A new context?

Reply:
I agree, it will take some more time, I'll update the PR.